pkg/state/kubernetes.go

```diff
@@ -0,0 +1,66 @@
+package state
```

Comment:
How come we need to have a noop implementation for kuberenetes in the state package?

Reply:
Was that when it's trying to create a Client? If so, we can probably avoid creating the Client entirely as it's not needed